README.md

```diff
@@ -1,35 +1,23 @@
 # Remix Examples
 
-Welcome to `@remix-run/examples`!
+Welcome to `@remix-run/examples`, the **community-driven** repository.
 
-If you have an example you'd like to share, please submit a pull request!
+Here you'll find various examples of using Remix to accomplish certain tasks. 
 
-**This is a community-driven repository.**. Here you'll find various examples of using Remix to accomplish certain tasks. Each example is a complete application including a build and even a button to preview a live instance of the app so you can play with it.
+Each example is a complete application including a build and even a button to preview a live instance of the app so you can play with it.
 
-Also, remember to check out the `README` for each example before getting started.
+Remember to check out the `README` for each example before getting started.
 
-You can also initialize a new project with any of these examples using the `--template` flag of the `create-remix` CLI. For example:
+You can also initialize a new project with any of these examples using the `--template` flag of the `create-remix` CLI. 
+
+For example:
 
 ```
-npx create-remix@latest --template https://github.com/remix-run/examples/tree/main/<example-dir>
+npx create-remix@latest --template examples/<example-dir>
```

Comment:
This will work again when https://github.com/remix-run/remix/pull/4208 is merged